src/components/UserInfo.js

```diff
@@ -1,19 +1,34 @@
 export default class UserInfo {
-  constructor({nameSelector, jobSelector}) {
+  constructor({ nameSelector, jobSelector, avatarSelector }) {
     this._nameElement = document.querySelector(nameSelector);
     this._descriptionElement = document.querySelector(jobSelector);
+    this._avatarElement = document.querySelector(avatarSelector);
   }
 
   getUserInfo() {
     this._userData = {
       name: this._nameElement.textContent,
-      description: this._descriptionElement.textContent,
-    }
-    return this._userData
+      about: this._descriptionElement.textContent,
+      avatar: this._avatarElement.src,
+    };
+    return this._userData;
   }
 
-  setUserInfo({ name, description }) {
+  setUserInfo({ name, about, avatar }) {
     this._nameElement.textContent = name;
-    this._descriptionElement.textContent = description;
+    console.log(about);
+    this._descriptionElement.textContent = about;
+    this._avatarElement.src = avatar;
   }
-}
\ No newline at end of file
+
+  getUserAvatar() {
+    const userAvatar = {
+      avatar: this._avatarElement.src,
+    };
+    return userAvatar();
+  }
+
+  setUserAvatar(avatar) {
+    this._avatarElement.src = avatar;
```

Comment:
`setUserAvatar` duplicates the code in line 12. Please, use it there or delete it